bld/CLMBuildNamelist.pm

```diff
@@ -4254,7 +4254,7 @@ sub check_input_files {
                 my $pathname = $nl->get_variable_value($group, $var);
                 # Need to strip the quotes
                 $pathname =~ s/['"]//g;
-
+                next if ($pathname =~ /UNSET$/);
```

Comment:
Yes, this is a good start. I'd also like it to not show up in the namelist, as having a file set to "$DIN_LOC_DIR/UNSET" is confusing and looks wrong. I'll likely work on this in addition to what you are doing here.

Reply:
I also did work in 017a7ed1ed to prevent blank files being listed. I think @jedwards4b is also a good protection although I think it really should trigger an error because something is screwy if you have a namelist item set to UNSET. 

The comment that generated this was...

https://github.com/ESCOMP/CTSM/issues/1198#issuecomment-833615077